packages/cf-icons/src/cf-icons.less

```diff
@@ -11,31 +11,46 @@
 // Size variables
 //
 
-// Icon height matches the 19px rendered canvas of text set in Avenir Next
-// sized at 16px ( 19/16 = 1.1875 )
+// Icon SVGs viewbox is 1200 (h) x 1000 (w).
+// The height matches the 19px rendered canvas of text set in Avenir Next
+// sized at 16px (19/16 = 1.1875).
 @cf-icon-height: 1.1875em;
 
+//
+// Embedded inline SVG data URLs.
+//
+
+// Mixin to inject an SVG from the ./icons/ directory
+// into a background-image property.
+@plugin "cf-icons-svg-inline";
+.u-svg-inline-bg( @type ) {
+    @svg: cf-icons-svg-inline( @type );
```

Comment:
Trivial, but `@type` seems like a slightly odd choice for this parameter. I think `@name` would be a little more intuitive.

Reply:
Changed in https://github.com/cfpb/capital-framework/pull/938/commits/f91b0f5f1615028ebee714fb637193a38ba868ce